atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -148,86 +148,82 @@ def run(self):
+                    )
+
+            registry_image = image.copy()
+            registry_image.registry = self.registry['uri']
+            max_retries = DOCKER_PUSH_MAX_RETRIES
+
+            for retry in range(max_retries + 1):
+                if self.need_skopeo_push() or source_docker_archive:
+                    self.push_with_skopeo(registry_image, insecure, docker_push_secret,
+                                          source_docker_archive)
+                else:
+                    # OSBS2 TBD either use store manifest from ManifestUtil
+                    # or tag_imag from utils.image
+                    # we won't need pushing
+                    # self.tasker.tag_and_push_image(self.workflow.data.image_id,
+                    #                                registry_image, insecure=insecure,
+                    #                                force=True, dockercfg=docker_push_secret)
+                    pass
+
+                if source_docker_archive:
+                    manifests_dict = get_all_manifests(registry_image, self.registry['uri'],
+                                                       insecure,
+                                                       docker_push_secret, versions=('v2',))
+                    try:
+                        koji_source_manifest_response = manifests_dict['v2']
+                    except KeyError as exc:
+                        raise RuntimeError(
+                            f'Unable to fetch v2 schema 2 digest for {registry_image.to_str()}'
+                        ) from exc
+
+                    wf_data.koji_source_manifest = koji_source_manifest_response.json()
+
+                digests = get_manifest_digests(registry_image, self.registry['uri'],
+                                               insecure, docker_push_secret)
+
+                if (not (digests.v2 or digests.oci) and (retry < max_retries)):
+                    sleep_time = DOCKER_PUSH_BACKOFF_FACTOR * (2 ** retry)
+                    self.log.info("Retrying push because V2 schema 2 or "
+                                  "OCI manifest not found in %is", sleep_time)
+
+                    time.sleep(sleep_time)
+                else:
+                    break
+
+            pushed_images.append(registry_image)
+
+            if not config_manifest_digest and (digests.v2 or digests.oci):
+                if digests.v2:
+                    config_manifest_digest = digests.v2
+                    config_manifest_type = 'v2'
+                else:
+                    config_manifest_digest = digests.oci
+                    config_manifest_type = 'oci'
+                config_registry_image = registry_image
 
             if config_manifest_digest:
-                push_conf_registry.config = get_config_from_registry(
-                    config_registry_image, registry, config_manifest_digest, insecure,
+                self.registry['config'] = get_config_from_registry(
```

Comment:
we don't need this anymore, as you will query for config when needed

also self.registry is just plugin variable adding config there won't change anything as self.workflow.conf.registry
is returning always new dict

Reply:
it was a remnant of previous revision. removed it.